SingularityUI/app/components/taskSearch/TaskSearchForm.cjsx

```diff
@@ -0,0 +1,100 @@
+Utils = require '../../utils'
+
+FormField = require '../common/input/FormField'
+DropDown = require '../common/input/DropDown'
+DateEntry = require '../common/input/DateEntry'
+Enums = require './Enums'
+Header = require './Header'
+
+TaskSearchForm = React.createClass
+
+    updateCount: (event) ->
+        @props.updateCount(event.target.value)
+
+    render: ->
+        <div className='col-xs-5'>
+            <Header
+                global = @props.global
+                requestId = @props.requestId
+            />
+            <form role="form" onSubmit={@props.handleSubmit}>
```

Comment:
You have a table component to generate tables for you - have you considered doing the same for forms? Even creating a wrapper component for each form group that contains a label and a `FormField` would be helpful